docs/upgrade-check.md

```diff
@@ -0,0 +1,22 @@
+# Upgrade and Security Bulletin Checks
```

Comment:
@soerenmartius the docs is based on [this](https://developer.hashicorp.com/terraform/cli/commands#upgrade-and-security-bulletin-checks).

Reply:
looks good to me! thanks!